db.sql

```diff
@@ -0,0 +1,47 @@
+-- Create database
+CREATE DATABASE twd_database;
+
+-- Connect to the database
+\c twd_database;
+
+
+-- Create a 'topics' table
+CREATE TABLE topics (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    email VARCHAR(255) NOT NULL,
+    topic VARCHAR(255) NOT NULL,
+    summary TEXT,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Create a 'projects' table
+CREATE TABLE projects (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    description TEXT,
+    link VARCHAR(255),
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+-- Assuming you already have a 'users' table for the votes connection
```

Comment:
@Nomad-Rob We won't be maintaining a central users table for this entire application, it'll all be handled through a managed IDP so the tie to users here should either record the users' email as their unique identifier in a flat structure (e.g. with a `user_email` field on the vote) or with a simple user pivot table of user_vote and a simple `users` table that is just an email column and an incrementing PK.